frontend/src/core/hotkeys/hotkeys.ts

```diff
@@ -239,6 +239,11 @@ const DEFAULT_HOT_KEY = {
     group: "Editing",
     key: "F2",
   },
+  "cell.duplicateSelection": {
+    name: "Duplicate selection",
+    group: "Editing",
+    key: "Ctrl-Shift-ArrowDown",
```

Comment:
```suggestion
    key: "Ctrl-Shift-ArrowDown",
    editable: false,
```

since right now we don't override it in the codemirror settings, this needs to be set to not editable

Reply:
That makes sense - added the two lines and set them both to noneditable.